app/views/user_registrations/user_registration_reminder.html.slim

```diff
@@ -0,0 +1,22 @@
+p
+  => "Dear"
```

Comment:
can we use email templates here? so organizers will be able to edit message from the website?
you need to add here https://github.com/LearnITGirl/WebPlatform/blob/master/app/models/email_template.rb#L2 new keys like `unregistered_mentee: 13` and `unregistered_mentor: 14`. Add new cases here https://github.com/LearnITGirl/WebPlatform/blob/master/app/models/email_template.rb#L26 with correct scopes, create templates (you can use /email_templates/new) and put them in the seed. if you won't have time for this let me know and i'll do it :)